src/main/scala/tech/sourced/gemini/Gemini.scala

```diff
@@ -54,7 +55,7 @@ class Gemini(session: SparkSession, keyspace: String = Gemini.defautKeyspace) {
   }
 
   def save(files: DataFrame): Unit = {
-    println(s"Writing ${files.rdd.countApprox(10000L)} files to DB")
+    log.debug(s"Writing ${files.rdd.countApprox(10000L)} files to DB")
```

Comment:
Imho this would be useful as a .`info`

Reply:
I had doubts about this one. Thought it's more like debug information, but now I have changed my mind. Fixed!